as_impl/src/main/java/ru/mano/aviasales/entity/Route.java

```diff
@@ -0,0 +1,79 @@
+package ru.mano.aviasales.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Route {
+    private int userId;
+    private List<Ticket> tickets;
```

Comment:
U can create new ArrayList() without constructor, so private List<Ticket> tickets = new ArrayList()... and... all ur created Route already will have initiated ArrayList;